src/screens/CreatePost/elements/CreatePollContainer/style/CreatePollContainerBaseStyle.js

```diff
@@ -144,7 +156,8 @@
   },
 
   bottombuttontext: {
-    fontFamily: fonts.inter[600]
+    fontFamily: fonts.inter[600],
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `color` property has been added to the `bottombuttontext` style. Ensure that this change doesn't affect other components that use this style and expect the color to be default.
```diff
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->